content/docs/integrations/kubernetes/index.md

```diff
@@ -55,7 +55,8 @@ Optional: Add a tag, Key: `3rdPartyIntegration` Value: `Roadie`
 
 6. Click ”Next Review”
 
-7. For the ”Role Name” enter suggested name: ”YOUR-COMPANY-NAME-backstage-backend-role-kubernetes”
+7. For the ”Role Name” enter: ”YOUR-COMPANY-NAME-roadie-read-only-role” (note: it is important to format the role as the following: ”YOUR-COMPANY-NAME-roadie-read-only-role”. If you do not, the role cannot be assumed. This is for security reasons.)
```

Comment:
Does YOUR-COMPANY-NAME need to be uppercase?

Reply:
No it is just a placeholder

Reply:
I think you should explain that it doesn't need to be uppercase.